resources/views/invoice/index.blade.php

```diff
@@ -1,28 +1,34 @@
-<table>
-    <thead>
-        <tr>
-            <th>ID</th>
-            <th>Customer</th>
-            <th>Date</th>
-            <th>Total</th>
-        </tr>
-    </thead>
-    <tbody>
-        @foreach($invoices as $invoice)
+@extends("layouts.main")
+
+@section("title", "Invoices")
+
+@section("content")
+    <table class="table">
+        <thead>
             <tr>
-                <td>
-                    {{$invoice->id}}
-                </td>
-                <td>
-                    {{$invoice->first_name}} {{$invoice->last_name}}
-                </td>
-                <td>
-                    {{$invoice->invoice_date}}
-                </td>
-                <td>
-                    ${{$invoice->total}}
-                </td>
+                <th>ID</th>
+                <th>Customer</th>
+                <th>Date</th>
+                <th>Total</th>
             </tr>
-        @endforeach
-    </tbody>
-</table>
\ No newline at end of file
+        </thead>
+        <tbody>
+            @foreach($invoices as $invoice)
+                <tr>
+                    <td>
+                        {{$invoice->id}}
+                    </td>
+                    <td>
+                        {{$invoice->first_name}} {{$invoice->last_name}}
+                    </td>
+                    <td>
+                        {{$invoice->invoice_date}}
+                    </td>
+                    <td>
+                        ${{$invoice->total}}
+                    </td>
+                </tr>
+            @endforeach
+        </tbody>
+    </table>
+@endsection
```

Comment:
View this PR without whitespace changes. Click the gear icon under "Files changed" and click "Hide whitespace".